app/assets/stylesheets/views/_search.scss

```diff
@@ -64,9 +59,9 @@ main.search {
       padding: 0 0 0 10px;
       display: block;
       height: 50px;
-      border-color: $border-colour;
+      border: 1px solid $border-colour;
       border-right: none;
-      line-height: 48px!important;
+      line-height: 48px !important;
```

Comment:
Is there a way of doing this without the !important? I know it's a legacy line, but it seems like a good opportunity to fix it. Unless this'll all change in the next round anyway, not sure what the plan is.

Reply:
I think this will get deleted when we switch to components, so I wasn't too concerned.